src/com/projectkorra/projectkorra/storage/DBConnection.java

```diff
@@ -38,7 +38,7 @@ public static void init() {
 				sql.modifyQuery(query);
 			}
 		} else {
-			sql = new SQLite(ProjectKorra.log, "[ProjectKorra] Establishing SQLite Connection.", "projectkorra.db", ProjectKorra.plugin.getDataFolder().getAbsolutePath());

```

Comment:
It was changed to "" because otherwise it outputs as "Establishing SQLite Connection.[SQLite] Connection established" all on one line, so it would be pointless to tell them that the connection is being established and that it's been established all in one message, and it would require re-working portions of the database code to make them output separately and actually be part of the two separate events, establishing the connection and confirming it's established.